routes/translations-1/translations-1.njk

```diff
@@ -0,0 +1,62 @@
+{% extends "base.njk" %}
+
+{% block content %}
+
+    <h1>{{ __('translations-1.title') }}</h1>
+
+    <div>
+        <p>
+            {{ __('translations-1.intro') }}
+        </p>
+
+
+        <form method="post" class="form">
+
+            <input type="hidden" name="_csrf" value="{{ csrfToken }}">
+
+            {% if data["partner_department_name"+language] !== "" %}
```

Comment:
We might be able to do all these lines with some sort of for loop over a list of strings? Something to poke at in a future PR if we're bored.